frameworks/Python/weppy/uwsgi.ini

```diff
@@ -0,0 +1,19 @@
+[uwsgi]
+master
+; Increase listen queue used for nginx connecting to uWSGI. This matches
+; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
+listen = 65535
```

Comment:
Please use 128.  uWSGI fails to start when net.core.somaxconn is not large enough.